src/components/cart.js

```diff
@@ -152,13 +155,27 @@ export default class Cart extends Component {
     // return this.props.client.fetchRecentCart();
   }
 
+  fetchMoneyFormat() {
+    return this.props.client.fetchShopInfo().then((res) => {
+      return res.moneyFormat;
+    });
+  }
+
   /**
    * initializes component by creating model and rendering view.
    * Creates and initalizes toggle component.
    * @param {Object} [data] - data to initialize model with.
    * @return {Promise} promise resolving to instance.
    */
   init(data) {
+    if (!this.config.moneyFormat) {
+      this.fetchMoneyFormat().then((moneyFormat) => {
+        this.moneyFormat = moneyFormat;
+        return moneyFormat;
+      }).catch((err) => {
+        return err;
```

Comment:
I don't think this catch is actually doing anything. Nothing is resolving this promise, and returning from a catch is technically rescuing the promise. Something should handle this error.